src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -72,17 +72,14 @@ def validate_request_update_args(request_args, config, reqmgr_db_service, param)
     
     TODO: rasie right kind of error with clear message 
     """
-
-    request_name = request_args["RequestName"]
-    # this need to be deleted for validation
-    del request_args["RequestName"]
+    # this needs to be deleted for validation
+    request_name = request_args.pop("RequestName")
```

Comment:
I didn't worry because we were directly accessing that key before, an exception would have happened way before.
Anyways, this validate func always receive a RequestName key/value.